src/main/java/com/SAS/Controllers/sasApplication/SASApplication.java

```diff
@@ -39,7 +39,7 @@ public SASApplication() {
      */
 
     public boolean login(String username, String password){
-       return userController.isUserExist(username, password);
+        return userController.isUserExist(username, password);
     }
 
```

Comment:
i am asking to understand, where are we creating this user? in the ui?

Reply:
It can be done using the register function or by creation of the team owner (the different assets - player and coach).